app/components/Facepile.js

```diff
@@ -43,6 +108,16 @@ function renderDefaultAvatar(user: User) {
   return <Avatar user={user} src={user.avatarUrl} size={32} />;
 }
 
+const MenuWrapper = styled.div`
+  display: flex;
+  max-width: calc(${(MAX_AVATAR_DISPLAY + 1) * 24}px + 0.5em);
```

Comment:
The maximum number of items per row is equals to `MAX_AVATAR_DISPLAY + the counter item`, what do you think about it?

Reply:
@tommoor

Reply:
Hey, I think it needs to be more of a standard list so that additional info like name and last viewed can be included. This is one of the reasons the issue is marked as `design needed`. I can try and make sure that designs happen in the next week, hold onto this PR in the meantime.